source/stylesheets/mixins/_media_queries.scss

```diff
@@ -2,7 +2,7 @@
 $breakpoints: (
   phone: 600px,
   tablet: 768px,
-  desktop: 1150px,
+  desktop: 1205px,
```

Comment:
Actually, disregard that. It was supposed to change the design disruptors media query, not that one.